webpack/webpack.config.base.js

```diff
@@ -20,13 +11,15 @@ module.exports = function getBaseConfig(config) {
     new CleanWebpackPlugin(config.outputPath, { verbose: false }),
   ];
 
+  const relativeAssetsPath = path.relative(config.outputPath, config.assetsPath);
```

Comment:
This is no longer used?

Reply:
Already removed, before you posted your comment ;-)